operator/src/main/resources/scripts/utils_base.sh

```diff
@@ -0,0 +1,256 @@
+# Copyright (c) 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
```

Comment:
Add comment something like:

# Important: Functions defined in file this must be able to work with unknown shells,
#                   e.g. pure shell instead of bash, when UNKNOWN_SHELL has been set to true.